volontulo/templates/static_pages/faq-volunteers.html

```diff
@@ -6,7 +6,7 @@
     <h2>Często zadawane pytania przez <strong>wolontariuszy</strong></h2>
     <div class="media">
         <div class="media-left">
-            possible 40x40px image
+            <img src="https://placeholdit.imgix.net/~text?txtsize=14&amp;txt=Symbol+80x80px&amp;w=80&amp;h=80&amp;txttrack=0" alt="symbol">
```

Comment:
Is it working here? When I tried to access this image I had to replace &amp; with &.